lib/presentation/widget/login/login_widget.dart

```diff
@@ -264,11 +258,20 @@ class _LoginWidgetState extends State<LoginWidget> {
   bool _checkCredentialError(Failure failure) {
     if (failure is AuthenticationFailure) {
       return failure.authenticationException is BadCredentials ||
-          failure.authenticationException is UnknownError;
+          failure.authenticationException is UserLocked  ||
+          failure.authenticationException is UnknownError ;
     }
     return false;
   }
 
+  String _getCredentialErrorMessage(AuthenticationFailure authenticationFailure) {
+    var authenException = authenticationFailure.authenticationException;
+    if (authenException is UserLocked) {
+      return AppLocalizations.of(context).user_locked_message;
```

Comment:
You can use a three-person operator like this.
`authenException is UserLocked ? AppLocalizations.of(context).user_locked_message : AppLocalizations.of(context).credential_error_message`

Reply:
look better :) ty